sdk/include/reactos/wine/test.h

```diff
@@ -213,6 +213,26 @@ extern void __winetest_cdecl winetest_print(const char* msg, ...);
 # define S5(x) (x)
 #endif
 
+enum WHITELIST_ATTRIBUTES
+{
+    WLA_OK = 0,
+    WLA_BROKEN_WIN = 1,
+    WLA_BROKEN_ROS = 2,
+    WLA_FLAKY = 4
+};
```

Comment:
How do you intend to use this ?

Reply:
I wanted to specify different use cases, to be able to selectively disable the test on ReactOS only. I thought about only using it for ReactOS and instead rather get the tests fixed or commented out, that fail on Windows, but then there are still a bunch of wine-tests, that fail on Windows. So I will probably add some kind of "annotation" that specifies where it fails on, like `` <...> "Test fails on #ROS and #WIN2003``  or something. Suggestions how to do it are welcome.

Reply:
Can we add WLA_TODO_ROS ? Broken &  flaky don't enter into that category.

As for specifying it, I'd suggest using `#REACTOS_TODO` `#REACTOS_FLAKY` etc. in the spec string. No need to add another annotation format if you already have one.